From ef84730b0c5f0f2b1426c75af2aaf5849ed652b9 Mon Sep 17 00:00:00 2001
From: Quintin Hill <Quintin.Hill@arqiva.com>
Date: Fri, 12 Jan 2018 12:35:26 +0000
Subject: [PATCH 2/3] condor_utils: add Windows specific define to MyString.h

strcasecmp used in this header is not defined on Windows.
---
 src/condor_utils/MyString.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/condor_utils/MyString.h b/src/condor_utils/MyString.h
index 5ad87b5076..5c726e193c 100644
--- a/src/condor_utils/MyString.h
+++ b/src/condor_utils/MyString.h
@@ -25,6 +25,11 @@
 #include <string.h>
 #include <stdarg.h>
 #include <string>
+#ifdef WIN32
+#ifndef strcasecmp
+#define strcasecmp _stricmp
+#endif
+#endif
 
 class MyString;
 class MyStringSource;
-- 
2.14.3

